### PR TITLE
Styled artsit

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
change artist page and merge dev to fix conflict

**Review by LauraCourquin — APPROVED** [review]
It's good for me !

File: frontend/src/components/DemoProfile/style.jsx [hunk 1]
```diff
@@ -1,15 +1,16 @@
 import styled from "styled-components";
 
 export default styled.div`
-  margin: 2rem auto;
+  margin: 3rem auto 10rem auto;
```
Comment (LuigiMenez) [inline frontend/src/components/DemoProfile/style.jsx:4]: The second auto is useless

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
The second auto useless

**Pull request closed without merging**